File: modul 3/soal 5/PRAK205-2210817220020-Helena Rosalina.c
#include <stdio.h>

int main ()
{

int a,b,alas,t,k,l;

scanf("%d %d", &a,&b);
alas=sqrt ((b*b) - (a*a));
printf("alas = %d cm \n", alas);
t=a;
printf("tinggi= %d cm \n", t);
k=a+b+alas;
printf("keliling= %d cm \n", k);
l=(alas*a)/2;
printf("luas = %d cm^2 ", l);

}

File: modul 3/soal 5/PRAK205-2210817220020-Helena Rosalina.py
import math
a=int(input())
b=int(input())

alas=math.sqrt((b*b)-(a*a))
print("Alas = {} cm".format(round(alas)))
t=a
print("Tinggi = {} cm".format(t))
k=a+b+alas
print("Keliling = {} cm".format(round(k)))
l=(alas*a)/2
print("Luas = {} cm^2 ".format(round(l)))